include/aspect/material_model/visco_plastic.h

```diff
@@ -75,6 +75,23 @@ namespace aspect
         std::vector<double> yielding;
     };
 
+
+    /**
+       * A data structure with the output of calculate_isostrain_viscosities.
+       */
+    struct CalculateIsostrainViscosities
```

Comment:
Suggest renaming to IsostrainViscosities. This structure doesn't do any calculations.